src/components/Molecules/SearchInput/SearchInput.js

```diff
@@ -32,13 +32,12 @@ const SearchInput = ({ onChange, placeholder, value, ...rest }) => {
           </SearchWrapper>
           <ActionWrapper>
             <ActionButton
+              as="button"
               showLabel={false}
               disabled="disabled"
+              aria-disabled="true"
               type="submit"
-              id=""
-              name="action"
-              placeholder=""
-              label=""
+              aria-label="Disable search button"
```

Comment:
I think this should be "Search button disabled", this makes it sound like you press the button to disable search.